bin/xo

```diff
@@ -1,41 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Does the `sawtooth` metapackage include the xo cli?  If not, removing `/bin/xo` from sawtooth-core breaks the App Dev Guide tutorial "Playing with the XO Transaction Processor" (and any other procedures that use the xo command). For example, see 

See https://github.com/hyperledger/sawtooth-core/blob/master/docs/source/app_developers_guide/intro_xo_transaction_family.rst

The step to install the `sawtooth` metapackage is in "Using Ubuntu for your Application Development Environment" (https://github.com/hyperledger/sawtooth-core/blob/master/docs/source/app_developers_guide/ubuntu.rst).

Reply:
The meta-package includes XO. https://github.com/hyperledger/sawtooth-core/blob/master/ci/ns-control#L28

Reply:
Great - thanks for the info.